tests/integration/conftest.py

```diff
@@ -52,6 +52,35 @@ def analysis() -> Analysis:
     return analysis
 
 
+@pytest.fixture
+def analyses() -> list[Analysis]:
+    analyses: list[Analysis] = []
+    for i, priority in enumerate(PRIORITY_OPTIONS):
```

Comment:
These numbers don't matter, the only point they have in the fixture is that they are unique. Seems more confusing to indicate that it is somehow important that the index is for the priority/type/status.

Will leave as is since it the variables do not carry meaning and it is only a fixture

Reply:
Can'¨t we skip the index which. uses a bad pattern and just generate a random number instead?

Reply:
Yes, changed it 👍